packages/slice-machine/src/css/modal.css

```diff
@@ -19,7 +19,9 @@ body {
 }
 
 /* Review modal animation   */
-
+.ReactModal__Overlay {
+  z-index: 1;
```

Comment:
what's this for?

Reply:
this was previously made for one single modal, I needed it too so generalized to all modals